docs/book/src/user/quick-start.md

```diff
@@ -158,6 +158,23 @@ Test to ensure the version you installed is up-to-date:
 clusterctl version
 ```
 
+{{#/tab }}
+{{#tab windows}}
+
+#### Install clusterctl binary with curl on windows
+Go to the working directory where you want clusterctl downloaded.
+
+Download the latest release; on windows, type:
```

Comment:
I wonder if we should explicitly state to run this in PowerShell. I don't believe this will work if someone uses cmd.exe.

Reply:
That looks great, thanks! I think that would help avoid possible confusion later on if someone is not used to using PowerShell as their default console on Windows.

I agree with Killian, let's use proper casing for these: "Windows using PowerShell".

Reply:
on it

Reply:
> That looks great, thanks! I think that would help avoid possible confusion later on if someone is not used to using PowerShell as their default console on Windows.
> 
> I agree with Killian, let's use proper casing for these: "Windows using PowerShell".

Done :v: